VirtoCommerce.InventoryModule.Web/Scripts/blades/fulfillment-center-detail.js

```diff
@@ -37,10 +37,12 @@ angular.module('virtoCommerce.inventoryModule')
 
         function saveChanges() {
             blade.isLoading = true;
-            fulfillments.update({}, blade.currentEntity, function (center) {
-                blade.currentEntityId = center.id;
-                blade.refresh(true);
+            fulfillments.update({}, [blade.currentEntity], function (centers) {
```

Comment:
Do we need `{},` first argument in the update method call?

Reply:
This is uri params. Yes, in the PUT action we can live without it.